logrotate.c

```diff
@@ -1511,6 +1512,18 @@ static int findNeedRotating(const struct logInfo *log, unsigned logNum, int forc
                             state->lastRotated.tm_hour, state->lastRotated.tm_min);
                 }
                 break;
+            case ROT_MINUTES:
+                if (((intmax_t)difftime(nowSecs, mktime(&state->lastRotated)) / 60) > log->minutes) {
+                    state->doRotate = 1;
+                }
+                if (!state->doRotate) {
+                    message(MESS_DEBUG, "  log does not need rotating "
+                            "(log has been rotated at %d-%02d-%02d %02d:%02d, "
+                            "which is less than %u minutes ago)\n", 1900 + state->lastRotated.tm_year,
```

Comment:
The message says `less than X minutes ago` but the logic (`>`) is actually `less than or equal X minutes ago`.

Reply:
done, thanks for the review.